glib/src/construct_cell.rs

```diff
@@ -12,8 +12,15 @@ use std::sync::RwLock;
 macro_rules! define_construct {
     (@common $ident:ident, $container:ident, $inner:ty) => {
         // rustdoc-stripper-ignore-next
-        /// Wrapper around a container type to be used for custom glib properties which get set during
-        /// the construct phase. This is especially useful when used alongside the Properties macro.
+        /// Experimental.
+        /// This type is mostly useful when used alongside the `glib::Properties` macro, when you want to create a property
+        ///  of type `Option<T>` and
+        /// - `T` doesn't implement `glib::value::ToValueOptional`.
```

Comment:
I wouldn't support that. If `T` doesn't implement that then `T` doesn't support being `NULL`